siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -1254,9 +1254,9 @@ def _do_callback_rxlock(self, pvname, value):
             # specifically for delta subsector (10), consider a failure only if
             # redundancy timing path is also not locked
             trigsrc = _LLTimeSearch.get_fout2trigsrc_mapping()[devname]
-            for out in outs_in_failure:
-                trigsrcout = trigsrc[f'OUT{out}']
-                if trigsrcout.sub[0:2] == '10':
+            if len(outs_in_failure) == 1:
+                out = outs_in_failure.pop()
+                if trigsrc[f'OUT{out}'].sub[0:2] == '10':
```

Comment:
I think this should have a comment mentioning how `len == 1` is a valid condition because we have only one crate with redundancy

Reply:
idem